src/name-generator.ts

```diff
@@ -0,0 +1,5 @@
+export class NameGenerator {
+  public getName(): string {
+    return 'Sofie TV 2 server'
```

Comment:
Should move this string to a constant, this is kind of sloppy Anders. Seriously dude.